subprojects/platform-native/src/main/java/org/gradle/nativeplatform/tasks/InstallExecutable.java

```diff
@@ -149,6 +151,7 @@ public RegularFileProperty getInstalledExecutable() {
     @SkipWhenEmpty
     @Nullable
     @Optional
+    @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
We shouldn't use `RELATIVE` with a single file, ever, no?

Reply:
Changed to `NAME_ONLY`.